src/rwv/ui/plot_widget.py

```diff
@@ -75,52 +84,94 @@ def __init__(self, window, db):
-        self.loc_checkbox = QtWidgets.QCheckBox("LOC", self)
-        # Set default value to true
-        self.loc_checkbox.setChecked(True)
-        # Connect our redraw function to the selector
-        self.loc_checkbox.stateChanged.connect(
-            lambda checked: self.canvas.redraw_points(JudgeCallType.LOC, checked)
-        )
-
         # Initialize UI values and graph
         self.init_interface_for_race()
 
         # widget layout
         layout = QtWidgets.QVBoxLayout()
-
-        # Checkbox layout
-        button_layout = QtWidgets.QHBoxLayout()
-        button_layout.addWidget(self.bent_knee_checkbox)
-        button_layout.addWidget(self.loc_checkbox)
-
-        layout.addWidget(close_db_button)
-        layout.addWidget(toolbar)
         layout.addWidget(self.race_label)
         layout.addWidget(self.race_combo_box)
         layout.addWidget(self.max_loc_label)
-        layout.addWidget(self.max_loc_combo_box)
+        layout.addWidget(self.max_loc_text_box)
         layout.addLayout(selector_layout)
-        layout.addLayout(button_layout)
-        layout.addWidget(self.canvas)
-
-        self.save_button = QtWidgets.QPushButton("Save Graph", self)
-        layout.addWidget(self.save_button)
-        self.save_button.clicked.connect(lambda: self.save_current_graph())
 
         # Tell widget to use specified layout
         self.setLayout(layout)
 
+        self.create_canvas_window()
+
+    def create_canvas_window(self):
+        """
+        Creates window that displays the generate chart.
+        """
+        self.canvas_window = QtWidgets.QWidget()
+        self.canvas_window.setWindowTitle("Endurance")
+        self.canvas_window.closeEvent = self.close_application
+
+        layout = QtWidgets.QVBoxLayout(self.canvas_window)
+        layout.addWidget(self.toolbar)
+        layout.addWidget(self.canvas)
+
+        self.canvas_window.setLayout(layout)
+
+        self.canvas_window.show()
+
+    def close_application(self):
+        """
+        Closes both of the windows once one is closed.
+        """
+        self.canvas_window.close()
+        self.close()
+
+    def create_menu_bar(self):
+        """
+        Creates and shows the menu bar. This contains actions for users relating to saving, closing the DB, and
+        toggling the display of Bent Knee and LOC on the graph.
+        """
+        # Initialize the menu bar.
+        menu_bar = QtWidgets.QMenuBar(self)
+
+        # Initialize the File button on the menu bar.
+        file_menu = QtWidgets.QMenu("&File", self)
+        menu_bar.addMenu(file_menu)
+
+        # Action to close the database file.
+        close_current_db = file_menu.addAction("Close Current DB")
+        close_current_db.triggered.connect(lambda: self.window.reset())
+
+        # Action to save the graph.
+        save_graph = file_menu.addAction("Save Graph")
+        save_graph.triggered.connect(lambda: self.save_current_graph())
+        save_graph.setShortcut('Ctrl+S')
```

Comment:
If I was a user I'd expect to ctrl+S to save the plot when the plot is the active window. However right now the shortcut only works when the control window is the active window

Reply:
This issue seems to be Windows/Linux specific. Will work with @mtp75 to hopefully get this fixed

Reply:
This along with the UI issue for Windows/Linux will be fixed in a PR that will be created a little later.

Reply:
plz make a ticket for the fix u'll be working on